frontend/.env.docker

```diff
@@ -0,0 +1,2 @@
+REACT_APP_API_URL=http://localhost:8080
```

Comment:
Changing this value to another port did not work, other files has to be changed as well. We made it work earlier for the docker-compose user case (I don't remember the other files we changed). But the docker user case is not very useful on its own. It would be more useful for the non-docker use case, but since the non-docker use case is broken, I can't test it. When I run mvn and npm, the frontend reports:
```
Proxy error: Could not proxy request /favicon.ico from localhost:3000 to http://app-server:8080.
See https://nodejs.org/api/errors.html#errors_common_system_errors for more information (ENOTFOUND).

Proxy error: Could not proxy request /favicon.ico from localhost:3000 to http://app-server:8080.
See https://nodejs.org/api/errors.html#errors_common_system_errors for more information (ENOTFOUND).

Proxy error: Could not proxy request /favicon.ico from localhost:3000 to http://app-server:8080.
See https://nodejs.org/api/errors.html#errors_common_system_errors for more information (ENOTFOUND).
```

Reply:
The non docker use-case works fine. This message is expected maybe I can put it in the readme